app/src/routes/api/v2/area.router.js

```diff
@@ -222,7 +230,7 @@ class AreaRouterV2 {
         if (ctx.request.body.language) {
             lang = ctx.request.body.language;
         }
-
+        logger.info('\n\n\nisSaved', isSaved)
```

Comment:
not very expressive. Was this for testing?

Reply:
Thanks for the catch!